recipes/cvat-sdk/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "cvat-sdk" %}
+{% set version = "2.24.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cvat_sdk-{{ version }}.tar.gz
+  sha256: 68b1de058db8f4f378adf1181bdc6ef7cc49d744330b3bc37f87eaea3143567e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >={{ python_min }}
+    - attrs >=21.4.0
+    - packaging >=21.3
+    - pillow >=10.3.0
+    - platformdirs >=2.1.0
+    - tqdm >=4.64.0
+    - tuspy ==0.2.5
+    - typing_extensions >=4.2.0
+    - python-dateutil >=2.5.3
+    - setuptools >=21.0.0
+    - urllib3 >=1.25.3
+
+test:
+  imports:
+    - cvat_sdk
+  commands:
+    - pip check
+  requires:
+    - pip
+    - python {{ python_min }}
+
+outputs:
+  - name: cvat-sdk
+  - name: cvat-sdk-masks
+    requirements:
+      host: &id001
```

Comment:
?

Reply:
That's a yaml anchor for this section not to copy it below again. - Grayskull generated it.
https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

If you like to have the sections written out I can replace those references by copying.

Reply:
```bash
grayskull pypi --maintainers achimgaedke --extras-require-all --extras-require-split cvat-sdk
```

with grayskull 2.7.3 - not sure since when this uses anchors.